kafka/README.md

```diff
@@ -6,16 +6,36 @@ This document defines how to describe Kafka-specific information on AsyncAPI.
 
 ## Version
 
-Current version is `0.1.0`.
+Current version is `0.2.0`.
 
 
 <a name="server"></a>
 
 ## Server Binding Object
 
-This object MUST NOT contain any properties. Its name is reserved for future use.
+This object contains information about the server representation in Kafka.
+
+##### Fixed Fields
 
+Field Name | Type | Description | Applicability [default] | Constraints
+---|:---:|:---:|:---:|---
+`saslMechamism` | string | Specifies the [`sasl.mechanism`](https://kafka.apache.org/documentation/#consumerconfigs_sasl.mechanism) config property that must be specified by Kafka clients connecting to this server. <br>Example values include `GSSAPI`, `PLAIN`, `OAUTHBEARER`, `SCRAM-SHA-256`, `SCRAM-SHA-512`  | OPTIONAL [`GSSAPI`] | -
```

Comment:
Really the default value is `GSSAPI`? Isn't it `PLAIN`?

Reply:
Also, shouldn't we have something like a `NONE` value to disable the saslMechanism?

Reply:
I did that to mirror what Kafka does by default - which is GSSAPI

https://github.com/apache/kafka/blob/fc68c0fc9b1199b4eb63d9d486b012667c1d448f/clients/src/main/java/org/apache/kafka/common/config/SaslConfigs.java#L32

<img width="1188" alt="image" src="https://user-images.githubusercontent.com/1444788/109037344-86122080-76c2-11eb-9b9c-8853caef7f03.png">

But I think adding a NONE option for when using `kafka` instead of `kafka-secure` might make sense, and make a better default.

Reply:
Yeah. In any case, if we move this to the Security Schemes Object, we would not need the `NONE` value anymore. Sorry, for the messed-up review. I realized about the other comments after I made the review.

Reply:
no problem at all - it's been a useful thread, thanks to both you and @smoya !